test/api_groups_test.js

```diff
@@ -17,8 +17,8 @@ describe('/groups', function() {
       forename: 'Julien',
     };
 
-    user.id = yield* api.createUser(user);
-    yield* api.login(user.email, user.password);
```

Comment:
we were using yield* because exported functions in api_tooling were generators. But this lacked flexibility, for example it's very difficult to parallelize setup.

I didn't do the parallelization in this patch because some tests need to be fixed (they take some specific ids for granted) and it proved to not be that easy to do, so I prefer to get this in first to make it easier to write future patches.